src/manifest.json

```diff
@@ -13,12 +13,19 @@
   "background": {
     "service_worker": "background.js"
   },
+  "font": {},
   "content_scripts": [
     {
       "matches": ["https://*/search*"],
       "include_globs": ["*.google.*/*"],
       "js": ["content-script.js"],
-      "css": ["github-markdown.css", "styles.css"]
+      "css": ["github-markdown.css", "styles.css", "katex.css"]
+    }
+  ],
+  "web_accessible_resources": [
+    {
+      "resources": ["fonts/*"],
+      "matches": ["https://*/*"]
```

Comment:
Why is this necessary?

Reply:
I added these fonts in the plugin so that I don't have to visit google. If I don't import it directly, it will give me an error:

![image](https://user-images.githubusercontent.com/91773459/206670397-e472bf85-2698-4bd3-88a0-4d77e3baa58c.png)

It can also be introduced through a CDN, but that will be slower

Reply:
got it